app/containers/StaticFixator.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component, PropTypes } from 'react';
+
+class StaticFixator extends Component {
```

Comment:
@AlexeyBarkow just use bootstrap layout. StaticFixator looks really ugly: a lot of logic just only for header